history-of-quakers/modernized/24-chapter-22.adoc

```diff
@@ -88,14 +100,21 @@ And I declare that no foreign prince, person, prelate, state, or potentate, has,
 or ought to have any power, jurisdiction, superiority, preeminence, or authority,
 ecclesiastical or spiritual, within this realm.
 
+--
+
 Besides this they were obliged to subscribe also another,
 with respect to their orthodoxy, and for excluding Socinianism.
 
-I A. B. profess faith in God the Father, and in Jesus Christ his eternal Son,
+[.embedded-content-document.legal]
+--
+
+I, _A. B._ profess faith in God the Father, and in Jesus Christ his eternal Son,
 the true God, and in the holy Spirit, one God, blessed forevermore:
 and do acknowledge the holy Scriptures of the Old
 and New Testament to be given by divine inspiration.
 
+--
+
 And lest any Roman Catholic might make use of this declaration,
 there were required sufficient Protestant witnesses that the declarer was a tanf dissenter.
```

Comment:
I just happened to notice this scanning through, the word `tanf` here should be `Protestant`